arkane/statmech.py

```diff
@@ -226,6 +225,21 @@ def load(self, pdep=False, plot=False):
         species object.
         """
         path = self.path
+        directory = os.path.abspath(os.path.dirname(path))
+        if not os.path.isfile(path):
+            path = os.path.join(directory, path)
```

Comment:
I think we can get rid of these lines actually. Sorry, when we spoke earlier I was wrong about these lines.